plugins/cloudinary/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/plugin-cloudinary",
-  "version": "0.0.2-12",
+  "version": "0.1.0",
```

Comment:
Not sure what version convention is more convenient. I guess we can promote it to v1.0.0 if it's okay for you?